### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by joenilson**
Added spanish translation

### CODE REVIEW

File: src/Resources/translations/messages.es.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+nedac_sylius_minimum_order_value_plugin:
+  ui:
+    minimum_order_value: "Valor mínimo de pedido"
+    enabled: "Habilitado"
+    attention: "Atención"
+    minimum_not_yet_reached: "Lo sentimos, no puedes ordenar todavía. Hay un valor de pedido mínimo establecido en: %minimumOrderValue% <br/><br/>Debes agregar más artículos al carrito por un valor de %difference%, para continuar con el pago."
```
Comment (4c0n) [inline src/Resources/translations/messages.es.yaml]: It seems to be missing the newline character at the end of the file, would you mind adding it please?
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

File: composer.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.4",
-        "sylius/sylius": ">=1.7.9 <1.8.0"
```
Comment (4c0n) [inline composer.json]: Please leave Sylius 1.8 out of this. It is not related to the Spanish translation. There already is a PR for that, but currently the tests are broken for 1.7 and that needs to be fixed first.

**Comment by 4c0n** [comment]
@joenilson Please see comments above. I'd be happy to merge your translation after the comments are addressed.

File: composer.json (hunk 2 above)
Comment (joenilson) [reply composer.json]: Sorry, this change was for my local branch i will remove it.

**Comment by joenilson** [comment]
removing wrong branch PR

**Pull request closed without merging**